Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -702,6 +706,12 @@ static AccessibilityPreferences accessibilityPreferences()
 #if HAVE(MEDIA_ACCESSIBILITY_FRAMEWORK)
     addCFNotificationObserver(mediaAccessibilityPreferencesChangedCallback, kMAXCaptionAppearanceSettingsChangedNotification);
 #endif
+#if HAVE(POWERLOG_TASK_MODE_QUERY) && ENABLE(GPU_PROCESS)
+    m_powerLogObserver = [[NSNotificationCenter defaultCenter] addObserverForName:kPLTaskingStartNotification object:nil queue:[NSOperationQueue currentQueue] usingBlock:^(NSNotification *notification) {
+        if (auto* gpuProcess = GPUProcessProxy::singletonIfCreated())
+            gpuProcess->enablePowerLogging();
+    }];
+#endif // HAVE(POWERLOG_TASK_MODE_QUERY) && ENABLE(GPU_PROCESS)
```

Comment:
What about WebContent and Networking?

Reply:
> What about WebContent and Networking?

That is a good point. The power logging emitted from WebKit is only from CoreMedia frameworks, AFAIK, so I believe this is only needed in the GPU process.

Thanks for reviewing!